draft-ietf-webpush-protocol.xml

```diff
@@ -1137,6 +1147,12 @@ HEADERS      [stream 82] +END_STREAM
           This could force the user agent to serialize subscription requests to the push
           service.
         </t>
+        <t>
+         A user agent that monitors subscriptions for other clients might

```

Comment:
I think proxy for other client is an important use case and feature - but it is tricky, and should not be added this late. Maybe a separate draft, or next version ?